Back-end/src/routes/Control/control.controller.ts

```diff
@@ -162,8 +162,49 @@ export const getNextControls: RequestHandler = async (req, res) => {
     return res.status(200).send({ success: true, data:{ nextControlsFiltered }, message: 'Lista de controles obtenida de manera correcta' });
 }
 
+/**
+ * Función que maneja la petición de obtener una lista de los controles pasados
+ * @route Get /control/:idMother
+ * @param req Request de la petición, se espera que tenga el id de la madre
+ * @param res Response, retorna un un object con success:true, data:{ controls: [{},{}] } y un message: "String" de la lista de controles si todo sale bien
+ */
 export const getPassControls: RequestHandler = async (req, res) => {
+    const idMother = req.params.idMother;
+
+    //se valida el _id de la madre ingresada
+    if ( !Types.ObjectId.isValid( idMother ))
+        return res.status(400).send({ success: false, data:{}, message: 'ERROR: El id ingresado no es válido.' });
 
+    const mother = await Mother.findById( idMother );
+
+    //se valida la existencia de la madre en el sistema
+    if ( !mother )
+        return res.status(404).send({ success: false, data:{}, message: 'ERROR: La madre ingresada no existe en el sistema.' });
+
+    //obtenemos la fecha actual
+    const date = new Date();
+    
+    //se setean la hora, minuto, seg y miliseg
+    date.setUTCHours(3);
+    date.setUTCMinutes(0);
+    date.setUTCSeconds(0);
+    date.setUTCMilliseconds(0);
+
+    const dateFormat = DateToFormattedString(date);
```

Comment:
Que se hace con esta variable?

Reply:
Buscamos manera de convertir el DateUTC a string en formato yyyy/mm/dd y esa función fue la unica que funcionó. 
Habían algunos get -> data.getUTCMonth (por ejemplo) que retornaban lo que les solicitabas, pero no había ninguno que retornara en formato yyyy-mm-dd por ultimo, solo retornaban por ej: Mon Nov 16, of 2021 (o algo así creo que era).

Reply:
Estoy casi seguro que yo una vez hice una funcion parecida, la busco y te la muestro
Lo otro, hay una biblioteca de js que es para justamente eso, formatear fecha y es super completa, sirve para la fecha la hora, etc. Se llama momentjs